### PR TITLE
add hasMultipleRows to the input component

### PR DESCRIPTION
**Pull request by EugeniusGR**

### CODE REVIEW

File: frontend/src/components/common/input/input.tsx [hunk 1]
```diff
@@ -25,18 +26,28 @@ const Input: React.FC<Props> = ({
...
-        type={type}
-        placeholder={placeholder}
-        className={styles.input}
-      />
+      {hasMultipleRows ? (
+        <textarea
+          {...field}
+          placeholder={placeholder}
+          className={styles.input}
+        ></textarea>
```
Comment (ghost) [inline frontend/src/components/common/input/input.tsx]:
```suggestion
        />
```

File: frontend/src/components/common/input/input.tsx [hunk 2]
```diff
@@ -25,18 +26,28 @@ const Input: React.FC<Props> = ({
...
-        {...field}
-        type={type}
-        placeholder={placeholder}
-        className={styles.input}
-      />
+      {hasMultipleRows ? (
+        <textarea
+          {...field}
+          placeholder={placeholder}
+          className={styles.input}
```
Comment (ghost) [inline frontend/src/components/common/input/input.tsx]: remove resize css property for the textarea with this class

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by Denidzio — DISMISSED** [review]

File: frontend/src/components/common/input/styles.module.scss [hunk 3]
```diff
@@ -24,6 +24,18 @@
...
+  display: block;
+  padding: 11px 40px;
+  border: none;
+  border-radius: 7px;
+  outline: none;
+
+  &::placeholder {
+    color: rgb(40 40 40 / 50%);
+  }
+}
```
Comment (ghost) [inline frontend/src/components/common/input/styles.module.scss]:
```suggestion
.textarea {
resize: none;
}
```
and add it only for testarea

**Review by ghost — CHANGES_REQUESTED** [review]

File: frontend/src/components/common/input/styles.module.scss (hunk 3 above)
Comment (EugeniusGR) [reply frontend/src/components/common/input/styles.module.scss]: should I delete all styles? It looks kinda raw
image

File: frontend/src/components/common/input/input.tsx [hunk 4]
```diff
@@ -25,18 +26,28 @@ const Input: React.FC<Props> = ({
...
-        {...field}
-        type={type}
-        placeholder={placeholder}
-        className={styles.input}
-      />
+      {hasMultipleRows ? (
+        <textarea
+          {...field}
+          placeholder={placeholder}
+          className={styles.textarea}
```
Comment (ghost) [inline frontend/src/components/common/input/input.tsx]:
```suggestion
          className={getAllowedClasses(styles.input, styles.textarea)}
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**